beard/clustering/blocking.py

```diff
@@ -99,52 +158,38 @@ def _validate(self, X, blocks):
 
     def _fit(self, X, y, blocks):
         """Fit base clustering estimators on X."""
+        def _grouper(jobs, seq):
```

Comment:
Indeed. In this case however, I believe it is fine -- I am the author of `sklearn.ensemble.base._partition_estimators` :)